R/github-utils.R

```diff
@@ -1,24 +1,30 @@
-## suppress the warning about multiple remotes, which is triggered by
-## the common "origin" and "upstream" situation, e.g., fork and clone
-gh_tree_remote <- function(path) {
-  suppressWarnings(gh::gh_tree_remote(path))
+github_remotes <- function(path = proj_get()) {
+  remotes <- git_remotes(path)
+  if (length(remotes) == 0) return(NULL)
+  m <- vapply(remotes, function(x) grepl("github", x), logical(1))
+  if (length(m) == 0) return(NULL)
+  remotes[m]
+}
+
+github_origin <- function(path = proj_get()) {
+  r <- github_remotes(path)
+  if (length(r) == 0) return(NULL)
+  parse_github_remotes(r)[["origin"]]
 }
```

Comment:
We started using `gh::gh_tree_remote()` before usethis made so much direct use of git2r. gh actually parses the config file. Now that we use git2r so much, we might as get remote names and URLs this ourselves, through more conventional means.